### DISCUSSION TITLE
Assertion failed error

### DISCUSSION THREAD

**Discussion by Lexa013** [gildor2/UEViewer#287] [Q&A]
Hey, i tried to use your tool on DeadByDaylight but i'am getting an error when loading a folder.

![image](https://user-images.githubusercontent.com/48451588/197410186-0d9f3e7e-8b86-4013-8bb0-6e5cd48cc5e3.png)

Here is the notify.log

[notify.log](https://github.com/gildor2/UEViewer/files/9847303/notify.log)

I have 0 clue of what is causing this error, if you have any patch or answer please tell me.

Thank you in advance
Lexa

**Comment by RainyLave** [comment]
Hello, please check if you are using the wrong vision of the unreal engine, for me, when I use the right vision, the software works well, instead, when I use the wrong vision,  it does make the wrong output just like yours did. Hope it helps!
